.github/workflows/brand-plugin-test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       module-repo: ${{ github.repository }}
       module-branch: ${{ needs.setup.outputs.branch }}
       plugin-repo: 'bluehost/bluehost-wordpress-plugin'
+      plugin-branch: 'add/product-pages'
```

Comment:
We'll need to remember to remove this line before merging or at least before tagging a release.